components/gateway/cmd/main.go

```diff
@@ -17,10 +22,10 @@ import (
 )
 
 type config struct {
-	Address string `envconfig:"default=127.0.0.1:3001"`
+	Address string `envconfig:"default=127.0.0.1:3000"`
 
-	DirectorOrigin  string `envconfig:"default=http://127.0.0.1:3000"`
-	ConnectorOrigin string `envconfig:"default=http://127.0.0.1:3000"`
+	DirectorOrigin  string `envconfig:"default=http://127.0.0.1:3001"`
```

Comment:
why you changed this?

Reply:
I want to discriminate those 2 services.
According to this comment:
https://github.com/kyma-incubator/compass/pull/975#discussion_r393465002